content/content/oop_macro.md

```diff
@@ -55,7 +55,7 @@ macro class*(head, body: untyped): untyped =
     # --------------------
     # Ident !"Animal"
     typeName = head
-    baseName = ident("RootObj")
+    baseName = newIdentNode("RootObj")
```

Comment:
Why was this changed? I don't really have a problem with the change, but I'm curious wrt what your reasoning is.